srcpkgs/growlight/patches/1bfbad3daf23276167e28dedb73dcd1af4c9f826.diff

```diff
@@ -0,0 +1,43 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 9d85c10..4c1ad5f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -34,7 +34,7 @@ add_compile_options(-Wall -Wextra -W -Wshadow -Wformat -fexceptions)
+ find_package(PkgConfig REQUIRED)
+ find_package(Threads)
+ set_package_properties(Threads PROPERTIES TYPE REQUIRED)
+-find_package(Notcurses 2.1.8 CONFIG)
++find_package(Notcurses 2.2.6 CONFIG)
+ set_package_properties(Notcurses PROPERTIES TYPE REQUIRED)
+ pkg_check_modules(LIBATASMART REQUIRED libatasmart>=0.19)
+ pkg_check_modules(LIBBLKID REQUIRED blkid>=2.20.1)
+diff --git a/README.md b/README.md
+index c8ef069..ec250d2 100644
+--- a/README.md
++++ b/README.md
+@@ -22,7 +22,7 @@ Dependencies:
+  - libcryptsetup 2.1.5+
+  - libdevmapper 1.02.74+
+  - libnettle 3.5.1+
+- - libnotcurses 2.1.8+
++ - libnotcurses 2.2.6+
+  - libpci 3.1.9+
+  - libpciaccess 0.13.1+
+  - libreadline 8.0+
```

Comment:
Please omit this part of the patch.

Reply:
Why?

Reply:
They don't affect the build, so it's a smaller diff :)

Reply:
My diff is directly from the upstream repo, the code will be in the next release :)

Reply:
I am aware, just think we can avoid such a large diff easily. Since we are pulling in the patch ourselves, we don't have to use it in its entirety.